src/test/integrasjonstester/kotlin/no/nav/familie/ks/sak/kjerne/behandling/domene/BehandlingRepositoryTest.kt

```diff
@@ -0,0 +1,133 @@
+package no.nav.familie.ks.sak.no.nav.familie.ks.sak.kjerne.behandling.domene
+
+import no.nav.familie.ks.sak.OppslagSpringRunnerTest
+import no.nav.familie.ks.sak.data.lagBehandling
+import no.nav.familie.ks.sak.data.lagFagsak
+import no.nav.familie.ks.sak.data.randomAktør
+import no.nav.familie.ks.sak.kjerne.behandling.domene.Behandling
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingRepository
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingStatus
+import no.nav.familie.ks.sak.kjerne.behandling.domene.BehandlingÅrsak
+import no.nav.familie.ks.sak.kjerne.fagsak.domene.Fagsak
+import no.nav.familie.ks.sak.kjerne.fagsak.domene.FagsakRepository
+import no.nav.familie.ks.sak.kjerne.personident.AktørRepository
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.Assertions.assertNull
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.beans.factory.annotation.Autowired
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class BehandlingRepositoryTest : OppslagSpringRunnerTest() {
+
+    @Autowired
+    private lateinit var behandlingRepository: BehandlingRepository
+
+    @Autowired
+    private lateinit var fagsakRepository: FagsakRepository
+
+    @Autowired
+    private lateinit var aktørRepository: AktørRepository
+
+    private lateinit var fagsak: Fagsak
+    private lateinit var behandling: Behandling
+
+    @BeforeEach
+    fun beforeAll() {
```

Comment:
```suggestion
    fun beforeEach() {
```

Dette er egentlig en beforeEach (gitt annotasjon)